common-crafting.lic

```diff
@@ -181,4 +181,38 @@ module DRCC
       DRC.bput("stow my #{logbook} logbook", 'You put', 'What were you referring to')
     end
   end
+
+  def order_enchant(stock_room, stock_needed, stock_number)
+    stock_needed.times do
+      order_item(stock_room, stock_number)
+      stow_crafting_item(left_hand, @bag, @belt)
+      stow_crafting_item(right_hand, @bag, @belt)
+      next unless left_hand && right_hand
+    end
+  end
+
+  def fount(stock_room, stock_needed, stock_number, quantity)
+    case bput('tap my fount', /You tap .* inside your .*/, /You tap .*your .*/, /I could not find what you were referring to./)
```

Comment:
❓ Would using `DRCI.exists?` or `DRCI.inside?` simplify this?

Reply:
I would need to test.